.travis.yml

```diff
@@ -1,13 +1,22 @@
+dist: trusty
```

Comment:
This is not required IMO

Reply:
No it's not, it's just there to call out what list we are using to run docker.